source/Ansible-Protocol-091/Amqp091ExchangeDeclare.class.st

```diff
@@ -30,6 +30,10 @@ Amqp091ExchangeDeclare >> arguments: aValue [
 	arguments := aValue
 ]
 
+{ #category : #accessing }
```

Comment:
It has been deprecated, see https://github.com/ba-st/Ansible/blob/9f48c39e6813d26267a6f3e9f2d5e646e4f75b6a/assets/specs/amqp0-9-1.xml#L1256

I think it's a mistake I made originally and it was ported over to VA.